tasks/section5.yml

```diff
@@ -71,15 +79,19 @@
       - rule_5.1.4
 
 - name: "SCORED | 5.1.4 | PATCH | Ensure permissions on /etc/cron.daily are configured"
-  command: /bin/true
+  file:
+      dest=/etc/cron.daily
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md